Sieve of Eratosthenes

```diff
@@ -0,0 +1,40 @@
+#include <bits/stdc++.h> 
```

Comment:
This is not a standard header.
It is usually available on online programming platforms.
Please include all the required ones instead.